app/gas/management/commands/create_gas_history.py

```diff
@@ -0,0 +1,59 @@
+'''
+    Copyright (C) 2017 Gitcoin Core
+
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU Affero General Public License as published
+    by the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+    GNU Affero General Public License for more details.
+
+    You should have received a copy of the GNU Affero General Public License
+    along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+'''
+
+from django.core.management.base import BaseCommand
+from django.db import transaction
+from django.utils import timezone
+
+from dashboard.gas_views import lines
+from gas.models import GasHistory
+from gas.utils import gas_history
+
+
+class Command(BaseCommand):
+
+    help = 'generates some gas history objects for the gitcoin gas station'
+
+    def handle(self, *args, **options):
+        hour = int(timezone.now().strftime('%H'))
+        day = int(timezone.now().weekday())
+        print(hour, day)
+
+        # hourly all the time
+        # daily once a day
+        # weekly once a week
+        breakdowns = ['hourly']
+        if hour <= 0:
+            breakdowns.append('daily')
+            if day <= 0:
+                breakdowns.append('weekly')
+
+        with transaction.atomic():
+            for breakdown in breakdowns:
+                GasHistory.objects.filter(key__startswith=breakdown).all().delete()
+                for i, __ in lines.items():
+                    view = 'gas_history'
+                    key = f"{breakdown}:{i}"
+                    print(f"- executing {breakdown} {key}")
+                    data = gas_history(breakdown, i)
+                    print("- saving")
+                    GasHistory.objects.create(
```

Comment:
Maybe we can adjust this in the future to use `.bulk_create` - quick a dirty way to do this for now would be to add an empty list, drop `.objects.create`, append to the list, and `GasHistory.objects.bulk_create(list_of_gas_history_objs)`

Reply:
we only create 7 rows at a time right now.  i def agree that bulk create is better tho.

afc701c17e987afeffb7dede1193d291ffa185db